include/picongpu/param/incidentField.param

```diff
@@ -22,6 +22,8 @@
  * Configure incident field profile and offset of the Huygens surface for each boundary.
  *
  * Available profiles:
+ *  - profiles::DispersiveLaser<>     : Gaussian pulse allowing to set first-, second-, and third-order dispersion

```

Comment:
Well, your remark is absolutely correct. But from my point of view, both names are inaccurate. We simulate laser pulses, which is not reflected by either name. Would it be ok for you, if I rename all lasers pulses later, just before the new release? That is, 'GaussianBeam' -> 'GaussianPulse', 'DispersiveLaser' -> 'DispersiveGaussianPulse', etc.

However, this will break compatibility to old setups, and we should consider whether the renaming is worth breaking downward compatibility. Then again, with the change to incident field as the preferred laser initialization method and deprecation of the old method with the next release, we have such a break anyways, so it may be just the right time to do this renaming.

Reply:
If you agree, I will open an issue and assign to myself.

Reply:
I'm not sure what the best solution for the naming convention is, but it sounds like a good idea to generally refactor the names with the next release. For this PR the name `DispersiveLaser` might just be fine then.